docs/guides/installation.rst

```diff
@@ -1,36 +1,56 @@
 Installation
 ============
 
-This section describes how to install `HTSinfer` using Conda.
+This section describes how to install `HTSinfer`.
```

Comment:
I think it would make sense to mention (and maybe link?) both installation methods here. Also, given that both methods require Conda, it would make sense to point this out here (and also recommend Mamba usage). Something like this:

---

This section describes how to install HTSinfer, either from Bioconda (recommended for most users) or GitHub (for experienced users and developers).

Both installation methods require the popular package and environment manager [Conda](https://docs.conda.io/en/latest/miniconda.html) to be available on your system. 

> For improved performance, we recommend using Conda's drop-in replacement [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) instead. In that case, simply replace `conda` with `mamba` in all of the commands below.

Reply:
Great, added this (even linked it to the section!)